src/shared/components/Animations/ListItem.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import './list-item.less';
+
+export interface ListItemProps {
+  label: string;
+  projectNumber?: number;
```

Comment:
I'd remove that. Plus I guess this list can be anything, not necessarily an organisation list

Reply:
forgot to make this generic

Reply:
👍